src/shibuya.js

```diff
@@ -106,58 +107,79 @@ const main = async () => {
     // One-time setup - a proxy account needs to be created to execute an XCM message on behalf of its user
     // We also need to transfer tokens to the proxy account to pay for XCM and task execution fees
     console.log('\n1. One-time proxy setup on Shibuya');
-    console.log(`\na) Add a proxy for Alice on Shibuya If there is no proxy of Turing (paraId:${TuringDev.paraId}) \n`);
-    const proxyType = 'Any';
-    // const proxyType = 'DappsStaking';
-    const proxyOnShibuya = shibuyaHelper.getProxyAccount(TuringDev.paraId, shibuyaAddress);
+    console.log(`\na) Add a proxy for Alice If there is none setup on Shibuya (paraId:${TuringDev.paraId}) \n`);
+
+    const proxyTypeShibuya = 'Any'; // We cannotset proxyType to "DappsStaking" without the actual auto-restake call
+    const proxyOnShibuya = shibuyaHelper.getProxyAccount(shibuyaAddress, TuringDev.paraId);
```

Comment:
I change the order of parameter in this getProxyAccount, because it makes more sense to have the wallet address in the front.